client/src/pages/WaitingRoom.js

```diff
@@ -98,7 +98,6 @@ function WaitingRoom(props) {
 
   const handleClose = () => {
     history.push("/dashboard");
-    window.location.reload(false);
```

Comment:
Is this causing problems for the waiting room?

Reply:
It was added to solve the problem of unpopulated table when coming back from "waitinglist" route to "dashboard" route using history push. Then it got changed in such a way that the table gets reloaded on location pathname change. So I removed this

Reply:
We shouldn't be doing this actually, so if it is causing issues for waiting room, we should look at why